ui/lib/sync/addon/components/secrets/page/destinations/destination/details.hbs

```diff
@@ -0,0 +1,43 @@
+{{!
+  Copyright (c) HashiCorp, Inc.
+  SPDX-License-Identifier: BUSL-1.1
+~}}
+
+<Secrets::Page::Destinations::Destination::Header @destination={{@destination}} />
+
+<Toolbar>
+  <ToolbarActions>
+    <Hds::Button @text="Delete destination" @color="secondary" class="toolbar-button" />
```

Comment:
I assume permissions-based hiding will come later?

Reply:
That was my plan! But I'll confirm with Jordan 😄